local-cli/cliEntry.js

```diff
@@ -51,9 +51,13 @@ function printHelpInformation() {
     cmdName = cmdName + '|' + this._alias;
   }
 
+  const usage = this.usage();
+  const usageIsString = typeof(usage) === 'string';
+  const usageString = usageIsString ? usage : '[options]';  
```

Comment:
There's going to be an issue with this eventually.  For example, how does one add an examples section to the `link` command?  If you did add examples, you'd lose the original usage() value (`[options] [packageName]`).